src/api/checkout.js

```diff
@@ -7,13 +7,13 @@ app.post('/', (req, res) => {
     if (typeof req.body.studentId !== 'number') {
         return res.status(400).failureJson('A student id is required.');
     }
-    if (typeof req.body.itemAddresses !== 'object'
-        || typeof req.body.itemAddresses[0] !== 'string') {
+    if (typeof req.body.equipmentAddresses !== 'object'
+        || typeof req.body.equipmentAddresses[0] !== 'string') {
```

Comment:
Performance-wise, the effect is negligible.  Unless someone is checking out millions of resistors, this shouldn't be a problem.  In terms of the code itself, the only time this really comes up is when the student's checked out items are being displayed, and even that isn't too bad.  Having to store models completely differently than items seemed like too much overhauling of the current system, and not worth the time.  That being said, if we do go on to do overdue models, we may need to switch to this in the future.